src/writeData/components/fileUploads/CSV.md

```diff
@@ -0,0 +1,7 @@
+For more detailed and up to date information check out the <a href="https://docs.influxdata.com/influxdb/v2.0/write-data/developer-tools/csv/#csv-annotations" target="_blank" rel="noreferrer">Annotated CSV Documentation</a>
+
+##### Getting Started
+
+The CSV uploader is a simple tool that can be used to upload files directly to a bucket by uploading an annotated CSV file.
+
+If you're looking to upload Annotated CSV files into a specific bucket, simply select the bucket you want your data uploaded to and drag & drop your data into the dropzone below.
```

Comment:
Would love some design thoughts on the text

Reply:
Personally when writing instructions I aim for simple imperative statements and brevity above all else.  What you have works but just to give you my perspective I'd have written the above as follows (feel free to take as much or as little of this as you like).  The only bit that I might expand on is where exactly one selects the bucket if that's not obvious in context.

> The CSV uploader uploads your annotated CSV files directly to a bucket.
> 
> Select your desired bucket, then drag and drop your file below to begin uploading.